### PR TITLE
Added Bangla Language

### PR DESCRIPTION
**Pull request by Itsmemonzu**
Hope this helps, Please let me know your opinions by reviewing!

### CODE REVIEW

**Comment by andrei1058** [comment]
I'll be waiting for a complete translation, otherwise you can close this yourself. We do not need partial implementations.

**Comment by Itsmemonzu** [comment]
Okay, I'll make sure to translate it to full Bangla today!

**Comment by Itsmemonzu** [comment]
Okay, I'll make sure to translate it fully today!

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Bangla.java [hunk 1]
```diff
@@ -335,24 +335,24 @@ public Bangla() {
 
         //
         yml.addDefault(Messages.SHOP_INDEX_NAME, "&8Quick Buy");
-        yml.addDefault(Messages.SHOP_INSUFFICIENT_MONEY, "{prefix}&cYou don't have enough {currency}! Need {amount} more!");
-        yml.addDefault(Messages.SHOP_NEW_PURCHASE, "{prefix}&aYou purchased &6{item}");
+        yml.addDefault(Messages.SHOP_INSUFFICIENT_MONEY, "{prefix}&cApnar kache porjapto {currency} ney! Apnar aro {amount} beshi lagbe!");
+        yml.addDefault(Messages.SHOP_NEW_PURCHASE, "{prefix}&a&aApni &6{item} &akinlen.");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Bangla.java]: fix tis color code please

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Bangla.java [hunk 2]
```diff
@@ -183,7 +182,7 @@ public Bangla() {
         yml.addDefault(Messages.FORMATTING_SCOREBOARD_TEAM_ALIVE, "&a&l✓");
         yml.addDefault(Messages.FORMATTING_SCOREBOARD_NEXEVENT_TIMER, "mm:ss");
         yml.addDefault(Messages.FORMATTING_SCOREBOARD_YOUR_TEAM, "&7 YOU");
-        yml.addDefault(Messages.FORMATTING_ACTION_BAR_TRACKING, "&fTracking: {team} &f- Distance: {distance}m");
+        yml.addDefault(Messages.FORMATTING_ACTION_BAR_TRACKING, "&fTracking: {team} &f- duratto: {distance}m");
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/Bangla.java]: Please use upper case for duratto

**Review by andrei1058 — CHANGES_REQUESTED** [review]

**Pull request merged**